### DISCUSSION TITLE
Update git from 1.8.3.1 to 2.22.0

### DISCUSSION THREAD

**Discussion by LAIM0** [community/community#21668] [New to GitHub]
<p>Hello,</p>
<p>I have git on my CentOS 6 server, but it’s the version 1.8.3.1 and I want to have the latest version (2.22.0).</p>
<p>I did not find any clear guide, I turn to you to know if my future method of update is viable.<br>
I currently have several repositories, I understand that the command “git clone <a href="https://github.com/git/git" rel="nofollow noopener">https://github.com/git/git</a>” would allow me to update git, or is there any other manipulation?<br>
I would like to keep my current configuration and my repositories. I’m just trying to get the latest version of git without impacting my current system.</p>
<p>Are there any big changes between my version and the last one?<br>
Thanks for your help.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p><code>git clone [https://github.com/git/git](https://github.com/git/git)</code> will get you the source code for Git, but it won’t be anything that you can execute on your machine. You would have to build Git from source in order for that to happen.</p>
<p>From what I understand, <a href="https://wiki.centos.org/PackageManagement/Yum" rel="nofollow noopener">CentOS uses the yum package manager</a>. So you may want to check the instructions for how to use yum to upgrade the software installed on your CentOS machine.</p>
<p>And you can check the <a href="https://github.com/git/git/tree/master/Documentation/RelNotes" rel="nofollow noopener">Git release notes</a> for the things that have changed between v1.8.3.1 (released ~6 years ago) and v2.22.0. For the most part, you probably won’t notice many large differences. Some commands have improved. Some performance has been improved. But the latest version of git will work on your old repositories with no issue and no extra steps needed.</p>
<p>I hope that helps!</p>

**Comment by LAIM0** [comment]
<p>Hello lee-dohm,<br>
Thanks a lot for your help! Indeed CentOS allows the update by the way of yum directly, I did not know it.</p>
<p>(# sudo yum update git)<br>
Thanks for the information about the changes since 6 years ahah!<br>
Have a nice day!</p>

**Reply by Khnaz35** [reply]
How did you do that?
When am running the command i am getting 
```[centos@ip-172-XXX-XXX-XXX ~]$ sudo yum update git
Loaded plugins: fastestmirror
Loading mirror speeds from cached hostfile
 * base: download.cf.centos.org
 * extras: download.cf.centos.org
 * updates: download.cf.centos.org
No packages marked for update
[centos@ip-172-XXX-XXX-XXX ~]$
